tools/sof_perf_analyzer.py

```diff
@@ -228,13 +231,15 @@ def analyze_perf_info():
         perf_info.groupby('COMP_ID')['CPU_AVG_MCPS'].max(),
         perf_info.groupby('COMP_ID')['CPU_PEAK_MCPS'].min(),
         perf_info.groupby('COMP_ID')['CPU_PEAK_MCPS'].mean(),
-        perf_info.groupby('COMP_ID')['CPU_PEAK_MCPS'].max()
+        perf_info.groupby('COMP_ID')['CPU_PEAK_MCPS'].max(),
+        perf_info.groupby('COMP_ID')['CPU_PEAK'].mean() * CPC_MARGIN,
```

Comment:
You don't really want to add it here, you can assign a new column just like 'PEAK(MAX)/AVG(AVG)'

Reply:
@aiChaoSONG ， changed, please check again